src/openfl/_internal/renderer/opengl/GLRenderer.hx

```diff
@@ -122,13 +122,13 @@ class GLRenderer extends AbstractRenderer {
 	}
 	
 	
-	public function getMatrix (transform:Matrix):Array<Float> {
+	public function getMatrix (transform:Matrix, snapToPixel: Bool = false):Array<Float> {
```

Comment:
Personally, I'm not a fan of optional args - I think it's better to pass the value explicitly for readability (also so we don't forget to all the places where it might make sense, also saves some null checks at run-time).

Reply:
There are many usages of this method. Since this flag is not relevant for other usages except in bitmap I don't want to force the flag.

Reply:
It's just 12 usages, but oh well :)